lib/binding/kvdb_interface.c

```diff
@@ -44,7 +44,11 @@
 
 /* clang-format off */
 
-#define HSE_KVDB_COMPACT_MASK  (HSE_KVDB_COMPACT_CANCEL | HSE_KVDB_COMPACT_SAMP_LWM)
+#define HSE_KVDB_COMPACT_MASK       \
+    (   HSE_KVDB_COMPACT_CANCEL |   \
+        HSE_KVDB_COMPACT_SAMP_LWM | \
+        HSE_KVDB_COMPACT_FULL )
+
```

Comment:
This was missed at first.  I think we should move these `_MASK` flags to be next to the flag definitions so they won't be missed in the future.

Reply:
Only reason I didn't place them in flags.h header originally was that I didn't think they had much value to users. Doesn't bother me if they move to the public header though.

Reply:
Let's come to a resolution on this. I still have the same position.

Reply:
Fine as is.